modules/dr-restoring-cluster-state.adoc

```diff
@@ -232,6 +232,194 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
 
 .. Repeat this step for each lost control plane host that is not the recovery host.
 
+. Delete and recreate other, non-recovery, control plane machines, one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
```

Comment:
I'm a little confused by the wording in this sentence. I think the issue is the comma. Is the following sentence accurate?

'Delete and recreate other, non-recovery control plane machines, in succession. After these machines are recreated, a new revision is forced and etcd scales up automatically.' 

You can keep one by one, I don't have an issue with that.

Reply:
Removed the comma after _other_...